mltsp/custom_feature_tools.py

```diff
@@ -200,7 +200,11 @@ def call_custom_functions(features_already_known_list, all_required_params,
     """
     """
     # import the custom feature defs
-    from .custom_feature_scripts import custom_feature_defs
+    try:
+        from .custom_feature_scripts import custom_feature_defs
+    except ImportError:
+        sys.path.append("/home/copied_data_files")
```

Comment:
This is only intended for use inside Docker containers - I should add a check here to be sure that's the case before appending the path. We need to import the custom features module somehow, perhaps there's a better solution (mltsp is mounted as read-only, so we can't just copy it into the package directory).